docs/section11/Rule_11-10.md

```diff
@@ -0,0 +1,285 @@
+      - swh_type - one of the
+      - input_capacity (btu/hr)
+      - storage_volume (gallons)
+      - draw_pattern (VERY_SMALL, LOW, MEDIUM, or HIGH)
+      - capacity_per_volume (btu/hr/gallon)
+    - get the service water heating system: `swh_eq = get_object_by_id(service_water_heating_equipment_id,RMD)`
+    - get the type of tank: `tank_type = swh_eq.tank.type`
+    - get the type of heater: `swh_type = swh_eq.heater_type`
+    - the type will be one of the ServiceWaterHeaterTankOptions, which are more detailed than the four types we need.  Determine our type based on the detailed type.
+    - set the type to OTHER.  If the tank_type is an identifiable type, type will be overwritten to that type: `type = "OTHER"`
+    - if the tank_type is CONSUMER_INSTANTANEOUS, COMMERCIAL_INSTANTANEOUS, or RESIDENTIAL_DUTY_COMMERCIAL_INSTANTANEOUS, set the type to INSTANTANEOUS: `if tank_type in ["CONSUMER_INSTANTANEOUS", "COMMERCIAL_INSTANTANEOUS", "RESIDENTIAL_DUTY_COMMERCIAL_INSTANTANEOUS"]: type = "INSTANTANEOUS"`
+    - if the tank_type is CONSUMER_STORAGE or COMMERCIAL_STORAGE, set the type to STORAGE: `if tank_type in ["CONSUMER_STORAGE", "COMMERCIAL_STORAGE"]: type = "STORAGE"`
+    - get the fuel type of the equipment: `fuel_type = shw_eq.heater_fuel_type`
+    - set the type based on tank_type and fuel type: `if fuel_type == "ELECTRICITY":`
+      - `if tank_type == "INSTANTANEOUS": type = "ELECTRIC_RESISTANCE_INSTANTANEOUS"`
+      - `elsif tank_type == "STORAGE": type = "ELECTRIC_RESISTANCE_STORAGE"`
+    - `elsif fule_type == "NATURAL_GAS":`
+      - `if tank_type == "INSTANTANEOUS": type = "GAS_INSTANTANEOUS"`
+      - `elsif tank_type == "STORAGE": type = "GAS_STORAGE"`
+    - `elsif fule_type == "FUEL_OIL":`
+      - `if tank_type == "INSTANTANEOUS": type = "OIL_INSTANTANEOUS"`
+      - `elsif tank_type == "STORAGE": type = "OIL_STORAGE"`
+    - if the draw_pattern is given, get the draw pattern: `if shw_eq.draw_pattern != nil: draw_pattern = shw_eq.draw_pattern`
+    - otherwise, determine the draw pattern based off the first_hour_rating of the shw_eq: `else:`
+      - First get the first hour rating (gallons): `first_hour_rating = shw_eq.first_hour_rating`
+      - if the first_hour_rating is less than 18 gallons, draw_pattern is "VERY SMALL": `if first_hour_rating < 18: draw_pattern = "VERY_SMALL"`
+      - otherwise if the first_hour_rating is less than 51 gallons, draw pattern is LOW: `elsif first_hour_rating < 51: draw_pattern = "LOW"`
+      - otherwise if the first_hour_rating is less than 75 gallons, draw pattern is MEDIUM: `elsif first_hour_rating < 75: draw_pattern = "MEDIUM"`
+      - otherwise draw pattern is HIGH: `else: draw_pattern = "HIGH"`
+    - get the capacity of the SHW system (make sure to convert to btu/hr): `input_capacity = swh_eq.input_power`
+    - get the volume of the tank (make sure to convert to gallons): `storage_volume = swh_eq.distribution_system.tank.storage_capacity`
```

Comment:
There can be multiple tanks. You think it is safe to assume one? Do we think there should only be one more App G? I think this may be a grey area? Perhaps this should assume there could be multiple.

Reply:
context changed to each ServiceWaterHeatingEquipment